### PR TITLE
Change 'iotjs_jval_t*' function parameters and locals to 'iotjs_jval_t' under modules directory (part 2)

### PR DESCRIPTION
**Pull request by LaszloLango**
Related issue: #1201 

IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by yichoi** [comment]
@LaszloLango, please make your code tidy https://travis-ci.org/Samsung/iotjs/jobs/279433247

**Comment by LaszloLango** [comment]
@yichoi I've updated the PR.

File: src/modules/iotjs_module_spi.c [hunk 1]
```diff
@@ -174,12 +174,12 @@ static void iotjs_spi_release_buffer(iotjs_spi_t* spi) {
 
 
 static void iotjs_spi_set_configuration(iotjs_spi_t* spi,
-                                        const iotjs_jval_t* joptions) {
+                                        iotjs_jval_t joptions) {
```
Comment (hs0225) [inline src/modules/iotjs_module_spi.c]: Please change `joptions` to `&joptions` at 186 line.

Comment (LaszloLango) [reply src/modules/iotjs_module_spi.c]: Good catch. Fixed.

Comment (LaszloLango) [reply src/modules/iotjs_module_spi.c]: @hs0225 why SPI module is not included in atrik053 precommit test? Is it intended?

Comment (hs0225) [reply src/modules/iotjs_module_spi.c]: @LaszloLango No. I'll fix it.

**Pull request merged**